src/app/core/components/header/header.component.html

```diff
@@ -13,27 +13,19 @@
         <li *ngIf="!authState.isLoggedIn">
           <button (click)="login()" class="button-outline">Einloggen</button>
         </li>
-        <div *ngIf="userLoaded" class="user-icon" [matMenuTriggerFor]="menu">
-          {{user?.name?.charAt(0)}}
+        <div *ngIf="userLoaded">
+          <button class="button-primary">
+            <div class="button-wrapper" [routerLink]="['/archive']">
+              <p>
+                Archiv
+              </p>
+              <mat-icon aria-hidden="false" aria-label="Example home icon">
+                archive
+              </mat-icon>
+            </div>
+          </button>
```

Comment:
Might be an option to create a separate component for that.
Also rather sth for the future

Reply:
Definitely!